### ISSUE TITLE
A question about this LAB

### ISSUE THREAD

**Issue by YitaoYuan** [GeorgeMLP/datalab#1]
This LAB only calculates the number of binary operations (like add) in the text form, without considering other operations or loops, right?

**Comment by GeorgeMLP** [comment]
That's correct. In most problems conditionals and loops are not allowed (c.f., `datalab.pdf`). In the floating point problems they are allowed, and they are not counted in the number of operations.
